app/views/organizations/adopter_fosterer/external_form/index.html.erb

```diff
@@ -3,9 +3,25 @@
     <% if params[:dashboard] %>
       <p><%= t("organizations.adopter_fosterer.form_instructions.dashboard") %></p>
     <% else %>
-      <p><%= t("organizations.adopter_fosterer.form_instructions.index", organization_name: Current.tenant.name) %></p>
+      <p><%= t(
+          "organizations.adopter_fosterer.form_instructions.index",
+          organization_name: Current.tenant.name,
+        ) %></p>
     <% end %>
-    <%= link_to 'View pets', adoptable_pets_path, class: "btn btn-primary float-end" %>
+    <div class="d-flex justify-content-end gap-2 pb-2">
+      <% if user_signed_in? %>
+        <%= link_to " #{t("general.view")} #{t("general.dogs")}",
+        adoptable_pets_path(species: "dog"),
+        class: "btn btn-primary" %>
+        <%= link_to " #{t("general.view")} #{t("general.cats")}",
+        adoptable_pets_path(species: "cat"),
+        class: "btn btn-primary" %>
+      <% else %>
+        <%= link_to t("general.become_an_adopter"),
```

Comment:
Is it possible to reach this route without being authenticated?

Reply:
You are correct, this can be removed.